examples/gnrc_lorawan/Makefile

```diff
@@ -7,6 +7,7 @@ USEMODULE += gnrc_netdev_default
 USEMODULE += auto_init_gnrc_netif
 USEMODULE += gnrc_lorawan
 USEMODULE += gnrc_netapi_callbacks
```

Comment:
This is not needed anymore now, right?

Reply:
addressed in https://github.com/RIOT-OS/RIOT/pull/11022/commits/b87aa2acbd749bee733b29d6ec57844963de4916